### PR TITLE
Styling / Minor Wording Changes

### PR DESCRIPTION
**Pull request by moea**
@whilo This PR makes some site-wide styling changes.  There's some very minor wording & formatting changes to the invariant post.  I'll submit a separate PR for invariant-specific wording/flow changes - since I'm not an expert in the subject matter, it may be helpful to have me go through the post to make it a bit more accessible.  N.B. the PR is against `feature/invariant-post`.

### CODE REVIEW

File: _drafts/2019-06-03-invariant-intro.md [hunk 1]
```diff
@@ -83,10 +83,9 @@ pattern is totally decoupled and arbitrarily read scalable. [^1]
     is working on top of the [dat project](https://dat.foundation/) using its
     publish and subscribe system, but other variants on [IPFS](https://ipfs.io/)
     or [BitTorrent](http://www.bittorrent.org/) are also possible.
- 
 
-# Adding facts to Datopia
```
Comment (whilo) [inline _drafts/2019-06-03-invariant-intro.md:88]: I finally took some time to look https://en.wikipedia.org/wiki/Capitalization_in_English up. It is regularly driving me a bit crazy, it is even worse than in German, but I think I can at least now see who to stick to one consistent style.

File: _drafts/2019-06-03-invariant-intro.md [hunk 2]
```diff
@@ -13,16 +13,17 @@ Target audience: Technical, CTOs, programming expertise (non-functional), Datalo
 Novelty: describe concept well enough for Datalog interests
 -->
 
-<blockquote class="literal left"> "The principle of least action is the basic
+# Goals
+{% quote Principle of Least Action, Scholarpedia|http://www.scholarpedia.org/article/Principle_of_least_action) %}
```
Comment (whilo) [inline _drafts/2019-06-03-invariant-intro.md:17]: I find this the best example to capture the importance of invariants in constraining the dynamics of the world, but maybe it is too academic. It is definitely not something that is well-known by computer scientists.

**Review by whilo — APPROVED** [review]
The source code changes look good to me, but somehow my bundler is broken, so I need to fix it before I can give feedback on the layout changes. Comfortaa looks good to me as a font.

File: _drafts/2019-06-03-invariant-intro.md (hunk 1 above)
Comment (moea) [reply _drafts/2019-06-03-invariant-intro.md:88]: Yeah, I don't have much of a handle on it.  I think my heuristic reduces to something like "capitalize everything that's longer than two letters" ;)

Comment (moea) [reply _drafts/2019-06-03-invariant-intro.md:88]: About the bundler thing - yeah, mine was broken too, or maybe I forgot how to use it.  I ended up manually installing the gems pixyll depends on (github-pages & jekyll-compose).  Not a big Ruby guy.

**Pull request merged**